lib/sfrest/backup.rb

```diff
@@ -37,6 +37,54 @@ def create_backup(site_id, datum = nil)
       @conn.post(current_path, datum.to_json)
     end
 
+    def execute_backup(options = {})
+      api = options[:api]
+      rconn = options[:rconn]
+      acsf = options[:acsf]
+      site_nid = options[:site_nid]
+      name = options[:name]
+      components = options[:components]
+      user = options[:user]
+
+      case api
+      when 'rest'
+        execute_rest_backup(rconn, site_nid, name, components)
+      when 'drush'
+        execute_drush_backup(acsf, site_nid, name, components, user)
+      else
+        raise "Unsupported API: #{api}"
+      end
+    end
+
+    def execute_rest_backup(rconn, site_nid, name, components)
+      payload = {
+        'label' => name,
+        'callback_url' => 'http://www.example.com',
```

Comment:
Do we care about callback_url? Afaik, that's optional.

Reply:
No we dont care about it, just for testing sake I added it.

Reply:
Can we remove it then? We shouldn't have the factory call unwanted sites.